ShchyrbaOleksandr/HW8/task3_main.py

```diff
@@ -0,0 +1,11 @@
+from task3_func import *
+
+figure = str(input("Write the name of the figure (rectangle, triangle or circle) for which you want to calculate the area: "))
```

Comment:
do not use the function  str() because function input() returns object of string type